France-ioi/AlgoreaFrontend
# add path to group link in member list

## Done

Forward `path` as input to member list component. Use it to construct the group url in the member list.

## Tests

1) Go to a group member list (see links below)
2) Click on `Sub groups`
3) Hover any group name

- With the [production app](https://dev.algorea.org/en/#/groups/by-id/672913018859223173;path=52767158366271444/details/members), you won't see any path in the url
- With the [branch app](https://dev.algorea.org/branch/fix/member-list-group-path/en/#/groups/by-id/672913018859223173;path=52767158366271444/details/members), you should see the path in the url


Incidentally, fixes #650
